beacon.yaml

```diff
@@ -3175,41 +3320,78 @@ components:
           example: "BAM format"
 
     FilteringTermsResponse:
-      type: object
       description: |
-        Filtering terms available in this Beacon.
+        Response of a query over Beacon info. Use `InfoResponseContent` when querying the Beacon info endpoints, and `DatasetResponseContent` when querying the datasets endpoint.
+      type: object
       required:
-        - beaconId
-        - apiVersion
-        - filteringTerms
+        - meta
+        - response
       properties:
-        beaconId:
-          description: |
-            Identifier of the beacon, as defined in `Beacon`.
-          type: string
-        apiVersion:
-          description: |
-            Version of the API implemented by this Beacon.
-          type: string
-          example: 'v2.0'
-        filteringTerms:
+        meta:
+          $ref: '#/components/schemas/FilteringTermsResponseMeta'
+        response:
+            $ref: '#/components/schemas/FilteringTermsResponseContent'
+    FilteringTermsResponseContent:
+      type: object
+      required:
+        - results
```

Comment:
This field does not exist among the properties below.

Reply:
good catch, results has been removed from required